deploy/helm/templates/l7-envoyfilter.yaml

```diff
@@ -20,11 +20,33 @@ spec:
             port_value: 15019
         filter_chains:
         - filters:
-          - name: "envoy.filters.network.tcp_proxy"
+          - name: envoy.filters.network.http_connection_manager
             typed_config:
-              "@type": type.googleapis.com/envoy.extensions.filters.network.tcp_proxy.v3.TcpProxy
-              stat_prefix: main_interval
-              cluster: main_internal
+              "@type": type.googleapis.com/envoy.extensions.filters.network.http_connection_manager.v3.HttpConnectionManager
+              stat_prefix: kmesh
+              route_config:
+                name: default
+                virtual_hosts:
+                - name: default
+                  domains:
+                  - '*'
+                  routes:
+                  - match:
+                      prefix: "/"
```

Comment:
If we look at the new configdump from 1.22 waypoint this is changed to 

```
- match:
    connectMatcher: {}
```

Reply:
It is `connectMatcher` in 1.21 as well which is used to match HTTO CONNECTH request, refer: https://www.envoyproxy.io/docs/envoy/latest/api-v3/config/route/v3/route_components.proto#envoy-v3-api-field-config-route-v3-routematch-connect-matcher

Because we don't use HBONE, change it to path matching.

Reply:
Ah, ic. This is not needed for us